### PR TITLE
initial README.md details

### PR DESCRIPTION
**Pull request by trescube**
don't merge yet, work in progress
Fixes pelias/whosonfirst#28

Connected to #33

### CODE REVIEW

**Comment by trescube** [comment]
Fixed #33

**Comment by riordan** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -1 +1,42 @@
-# whosonfirst
+# Who's on First Data Importer
+
+whosonfirst is a tool used for importing [Who's On First data](https://whosonfirst.mapzen.com/) from local files into a Pelias ElasticSearch repository.
```
Comment (orangejulius) [inline README.md]: database might be a better word than repository. repository makes me think of version control things

**Comment by orangejulius** [comment]
One thing that could be added: a note that Nodejs 0.12 is required (not 0.10).

(we should probably do this across all our repos eventually)

**Comment by trescube** [comment]
good call

**Comment by dianashk** [comment]
Would be nice to add this script to the package.json so people can run it via npm, like...

`npm run download`

```
"scripts": {
     "start": "node import.js",
     "download": "node download_data.js",
     "test": "node test/test | tap-dot
   },		   },
```

**Comment by trescube** [comment]
I can do that

**Comment by orangejulius** [comment]
@dianashk :+1:

**Comment by dianashk** [comment]
This looks good. :ship:

**Pull request merged**